Gemfile

```diff
@@ -105,10 +105,11 @@ source "https://gems.diasporafoundation.org" do
 
   gem "rails-assets-highlightjs",                         "9.12.0"
   gem "rails-assets-markdown-it",                         "8.4.2"
-  gem "rails-assets-markdown-it-hashtag",                 "0.4.0"
   gem "rails-assets-markdown-it-diaspora-mention",        "1.2.0"
-  gem "rails-assets-markdown-it-sanitizer",               "0.4.3"
+  gem "rails-assets-markdown-it-footnote",                "3.0.3"
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `rails-assets-markdown-it-diaspora-mention` should appear before `rails-assets-markdown-it-hashtag`.

Reply:
It appears before ..-hashtag! Trouble with Gemfiles?

Reply:
The review-bot uses it's own Gemfile, so this can be ignored and probably also fixes that problem for the future